atoma-types/src/lib.rs

```diff
@@ -599,9 +642,19 @@ impl Text2ImagePromptParams {
     pub fn decode_only(&self) -> Option<String> {
         self.decode_only.clone()
     }
+
+    /// Sets the image path for `Text2ImageModelParams` variant
+    pub fn set_image_path(&mut self, path: String) {
+        self.img_path = Some(path);
```

Comment:
We can have either img_path or img2img, right? If so, it should be an enum between these two.

Reply:
This is correct, I agree this is better. However changing this now requires changing the already deployed contract.. Can I instead create a future issue for this ?